src/app/campaign/flight/flight-zones-form.component.ts

```diff
@@ -57,7 +57,7 @@ export const validateMp3 = (value: string): { [key: string]: any } | null => {
       </div>
     </fieldset>
     <div *ngIf="zones?.controls?.length < zoneOptions?.length">
-      <a mat-button routerLink="." (click)="onAddZone()"> <mat-icon>add</mat-icon> {{ addCreativeLabel }} </a>
```

Comment:
good call, I had just copied the previous zones markup, but buttons should be buttons.